src/ConnectivityRenderer.js

```diff
@@ -47,7 +47,7 @@ class ConnectivityRenderer extends Component<DefaultProps, Props, State> {
   }
 
   componentDidMount() {
-    NetInfo.isConnected.addEventListener('change', this.checkInternet);
+    NetInfo.isConnected.addEventListener('connectionChange', this.checkInternet);
```

Comment:
`connectionChange` event handler receives and object instead of a boolean, so we need to handle that inside `checkInternet`. Please refer to http://facebook.github.io/react-native/docs/netinfo.html#addeventlistener

Reply:
Since we're listening to the `Netinfo.isConnected` property the listener receives a boolean.  I can take some time later on to try to build in `ConnectionType` handling. I figured this simple PR should handle the deprecation warnings for the time being.

Reply:
Oh, so it seems the deprecation applies to `NetInfo.addEventListener('change', handler)`, but not to `NetInfo.isConnected.addEventListener('change', handler)` right?

If so, we wouldn't need to change anything, since the latter is not deprecated.

Reply:
Eh, it seems the deprecation warning throws on RN 0.48 if the 'change' event is used either way :man_shrugging:.  Up to you :smile:, I threw this up after reading this issue: https://github.com/rauliyohmc/react-native-offline/issues/36

Thanks for the great utility!

Reply:
yep the warning is definitely there!

Reply:
@rauliyohmc Thoughts?

Reply:
I will wait for one or two more newer RN versions, it's still a fresh deprecation. I don't want new comers using RN < 0.48 to realise the library doesn't work as expected, having to go to the docs and figure out they need to use a different library version. That's not the best DX imo.

Reply:
@rauliyohmc how long though? It is pretty annoying to have this warning across the app while developing =/

Reply:
Since 0.49 landed and 0.50 will be coming soon, let's merge this. @gtfargo could you please rebase and merge conflicts? As soon as this is ready, I'll release a new version and update the readme

Reply:
@rauliyohmc believe you're set.  forgot about this 😄